### PR TITLE
Making everything up to date!

### PR DESCRIPTION
**Pull request by hexereii**
Added in new banner/links/descriptions/etc!

### CODE REVIEW

File: hud-data/berryhud.json [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+    "gamemode": ["tournament", "comp", "cp", "halloween", "koth","mvm", "pass", "pl"]
+    "res": ["16/9"],
+    "key": ["value"]
+  },
+  "tags": ["competitive", "sixes", "highlander", "ultiduo"]
+  "repo": "https://github.com/hexereii/BerryHUD"
+  "hash": "799723ff71e6dee2ae5ddd481cc953bba153c6e5"
+  "prerelease": false
+  "verified": true
+  "resources":
```
Comment (rbjaxter) [inline hud-data/berryhud.json:53]: missing opening [ after `"resources":`

that line should be `"resources": [`

File: hud-data/berryhud.json [hunk 2]
```diff
@@ -1,12 +1,57 @@
...
+	"pl"
+	],
+    "res": 
+	[
+	"16/9"
+	],
+    "key": 
+	[
+	"value"
+	]
```
Comment (rbjaxter) [inline hud-data/berryhud.json:42]: you can delete these lines, and then remove the comma after the bracket on line 38

File: hud-data/berryhud.json [hunk 3]
```diff
@@ -1,12 +1,57 @@
...
+    "album": "https://imgur.com/a/Q7QQwY2"
   },
+  "flags": [
+  "menus", 
+  "scoreboards", 
+  "crosshairs", 
+  "minmode",
+  "materials", 
+  "customization"
+  ]
```
Comment (rbjaxter) [inline hud-data/berryhud.json:18]: this one's missing a comma after the bracket

File: hud-data/berryhud.json [hunk 4]
```diff
@@ -1,12 +1,57 @@
 {
   "name": "BerryHUD",
-  "author": "hexereii",
+  "author": "heks",
   "social": {
-    "album": "https://imgur.com/a/ldt9NXv"
+    "steam_profile": "76561198041335138",
+    "steam_group": "",
```
Comment (rbjaxter) [inline hud-data/berryhud.json:6]: I believe @mastercoms prefers empty lines like this to just be deleted, so you can just remove line 6

File: hud-data/berryhud.json [hunk 5]
```diff
@@ -1,12 +1,57 @@
...
+    "key": 
+	[
+	"value"
+	]
+  },
+  "tags": [
+  "competitive", 
+  "sixes", 
+  "highlander", 
+  "ultiduo"],
```
Comment (rbjaxter) [inline hud-data/berryhud.json:48]: just for formatting, you'll want to move the `],` down to its own line

File: hud-data/berryhud.json [hunk 6]
```diff
@@ -1,12 +1,57 @@
...
   "repo": "https://github.com/hexereii/BerryHUD",
-  "hash": "b5358f39041a11561b44de858a0a62f8edbcfc5e",
-  "resources": [
-    "berryhud-banner"
+  "hash": "f26da6b62b6a24a78f2cac8aa0b07f5ebc16b6a1",
+  "prerelease": false
+  "verified": true
+  "resources": 
+    "myhud-banner"
+    "cool-image-1"
```
Comment (rbjaxter) [inline hud-data/berryhud.json:55]: you'll want to remove this line

**Review by rbjaxter — CHANGES_REQUESTED** [review]

File: hud-data/berryhud.json [hunk 7]
```diff
@@ -1,12 +1,52 @@
...
+  "competitive",
+  "sixes", 
+  "highlander", 
+  "ultiduo"
+  ],
   "repo": "https://github.com/hexereii/BerryHUD",
-  "hash": "b5358f39041a11561b44de858a0a62f8edbcfc5e",
+  "hash": "f26da6b62b6a24a78f2cac8aa0b07f5ebc16b6a1",
+  "prerelease": false
+  "verified": true
```
Comment (rbjaxter) [inline hud-data/berryhud.json:48]: you can remove these two lines. I believe `prelease` defaults to false, and `verified` is an internal hud-db thing you don't want to have

**Comment by rbjaxter** [comment]
@mastercoms other than that last review, I believe everything looks good otherwise. You should probably still take a peep though just in case I missed something, since there were a handful of corrections that needed to be made from the initial commit

File: hud-data/berryhud.json [hunk 8]
```diff
@@ -1,12 +1,52 @@
...
   "resources": [
-    "berryhud-banner"
+    "berrybanner.png"
```
Comment (mastercoms) [inline hud-data/berryhud.json:50]: Remove .png

**Review by mastercoms — CHANGES_REQUESTED** [review]

**Review by rbjaxter — APPROVED** [review]

**Review by mastercoms — APPROVED** [review]

**Pull request merged**